app/services/catalog/approval_transition.rb

```diff
@@ -46,11 +46,13 @@ def submit_order
     def mark_canceled
       finalize_order
       @order_item.update_message("info", "Order #{@order_item.order_id} has been canceled")
+      Rails.logger.info("Order #{@order_item.order_id} request_id: #{@order_item.insights_request_id} has been canceled")
```

Comment:
@lindgrenj6 If we have to add the request_id for every message in the future someone might forget and we will miss the messages. Can we investigate why the container logger/cloud watch is not getting our messages and doing the right thing.